CMakeLists.txt

```diff
@@ -623,9 +623,23 @@ if(BUILD_LIBRETRO)
             set(LIBRETRO_EXTENSION "a")
         endif()
         set_target_properties(tic80_libretro PROPERTIES SUFFIX "_partial.a")
-        add_custom_command(TARGET tic80_libretro
+        include(CheckCSourceCompiles)
+        check_c_source_compiles(
+"#ifndef __MSDOS__
+#error \"Not DOS\"
```

Comment:
Wait, is DJGPP a DOS compiler?  Are you building TIC-80 on DOS?

Reply:
For DOS. Cross-compiling from Debian